client/my-sites/customer-home/cards/features/reader/index.tsx

```diff
@@ -40,6 +43,11 @@ const ReaderCard = ( props: ReaderCardProps ) => {
 	const selectedTab = queryParams.get( 'selectedTab' ) || DEFAULT_TAB;
 
 	const streamKey = buildDiscoverStreamKey( selectedTab, [ 'dailyprompt' ] );
+	const dispatch = useDispatch();
+
+	useEffect( () => {
+		dispatch( recordReaderTracksEvent( 'calypso_reader_discover_viewed' ) );
+	}, [] );
```

Comment:
Shouldn't `dispatch` be a dependency as it's referenced from within the hook?

Reply:
Oh, yes! Thanks for catching it!